### PR TITLE
Copy-editing overview.md

### PR DESCRIPTION
**Pull request by gdemelo**
Extensive grammar revisions for zeppelin/docs/usage/interpreter/overview.md

### What is this PR for?
The documentation at zeppelin/docs/usage/interpreter/overview.md contained numerous instances of bad grammar. This PR improves it.

### What type of PR is it?
Documentation

### CODE REVIEW

File: docs/usage/interpreter/overview.md [hunk 1]
```diff
@@ -112,43 +112,41 @@ interpreter.start();
...
 
-`ConfInterpreter` is a generic interpreter that could be used by any interpreters. The input format should be property file format.
-It can be used to make custom setting for any interpreter. But it requires to run before interpreter process launched. And when interpreter process is launched is determined by interpreter mode setting.
-So users needs to understand the ([interpreter mode setting ](../usage/interpreter/interpreter_bindings_mode.html) of Zeppelin and be aware when interpreter process is launched. E.g. If we set spark interpreter setting as isolated per note. Under this setting, each note will launch one interpreter process. 
-In this scenario, user need to put `ConfInterpreter` as the first paragraph as the below example. Otherwise the customized setting can not be applied (Actually it would report ERROR)
+`ConfInterpreter` is a generic interpreter that can be used by any interpreter. The input format should be the property file format.
+It can be used to make custom settings for any interpreter. However, it requires being run before the interpreter process is launched. When the interpreter process is launched is determined by the interpreter mode setting.
```
[the remark this answers is not in the document]
Comment (gdemelo) [reply docs/usage/interpreter/overview.md]: Ok, reworded.

File: docs/usage/interpreter/overview.md [hunk 2]
```diff
@@ -69,41 +69,41 @@ You may use parameters from the context of interpreter by add #{contextParameter
...
 
 <img src="{{BASE_PATH}}/assets/themes/zeppelin/img/screenshots/interpreter_setting_spark.png" width="500px">
 
 
 ## Interpreter binding mode
 
-Each Interpreter Setting can choose one of 'shared', 'scoped', 'isolated' interpreter binding mode.
-In 'shared' mode, every notebook bound to the Interpreter Setting will share the single Interpreter instance. In 'scoped' mode, each notebook will create new Interpreter instance in the same interpreter process. In 'isolated' mode, each notebook will create new Interpreter process.
+In the Interpreter Settings, one can choose one of the 'shared', 'scoped', or 'isolated' interpreter binding modes.
+In 'shared' mode, every notebook bound to the Interpreter Setting will share a single Interpreter instance. In 'scoped' mode, each notebook will create new a interpreter instance in the same interpreter process. In 'isolated' mode, each notebook will create new a interpreter process.
```
[the remark this answers is not in the document]
Comment (gdemelo) [reply docs/usage/interpreter/overview.md]: Thanks for catching this.

File: docs/usage/interpreter/overview.md [hunk 3]
```diff
@@ -25,25 +25,25 @@ limitations under the License.
...
 
-When you click the ```+Create``` button in the interpreter page, the interpreter drop-down list box will show all the available interpreters on your server.
+When you click the ```+Create``` button on the interpreter page, the interpreter drop-down list box will show all the available interpreters on your server.
 
 <img src="{{BASE_PATH}}/assets/themes/zeppelin/img/screenshots/interpreter_create.png" width="280px">
 
-## What is interpreter setting?
-Zeppelin interpreter setting is the configuration of a given interpreter on Zeppelin server. For example, the properties are required for hive JDBC interpreter to connect to the Hive server.
+## What are the interpreter settings?
+The interpreter settings are the configuration of a given interpreter on the Zeppelin server. For example, certain properties need to be set for the Hive JDBC interpreter to connect to the Hive server.
```
[the remark this answers is not in the document]
Comment (gdemelo) [reply docs/usage/interpreter/overview.md]: Ok. Done.

**Comment by felixcheung** [comment]
great, thx!

**Comment by felixcheung** [comment]
LGTM
merging if no more comment

**Comment by felixcheung** [comment]
this was merged? is github not syncing?

**Pull request closed without merging**